schedule/yast/maintenance/autoyast_create_hdd_maintenance_deregister_aarch64.yaml

```diff
@@ -0,0 +1,16 @@
+---
+name: autoyast_create_hdd_maintenance_aarch64
+AUTOYAST: autoyast_sle15/autoyast_scc_up.xml
```

Comment:
It's strange you choose this profile to do installation, normally autoyast_scc_up.xml is used for upgrade.

Reply:
This line should be removed, you have already added the AUTOYAST=autoyast_sle15/create_hdd/create_hdd_maintenance.xml.ep in testsuite.

Reply:
> It's strange you choose this profile to do installation, normally autoyast_scc_up.xml is used for upgrade.

It can be used at here. No problem.

Reply:
> This line should be removed, you have already added the AUTOYAST=autoyast_sle15/create_hdd/create_hdd_maintenance.xml.ep in testsuite.

It was generated by automatic, I didn't add this setting.